wikimedia/wikipedia-ios
# Send is_temp and user_is_temp to MEP

**Phabricator:** https://phabricator.wikimedia.org/T376403

### Notes
This PR sends `is_temp` and `user_is_temp ` to MEP if the user is in a temporary auth state. To match Android, we are true for these values if a central auth username cookie exists and they do not have an entry in the local credential store.

Reference:
https://github.com/wikimedia/apps-android-wikipedia/blob/main/app/src/main/java/org/wikipedia/auth/AccountUtil.kt#L37

### Test Steps
1. Change WMFLogging.h to log everything
2. Fresh launch on app, ensure you are logged out. Watch the payloads as you click around and confirm you see `is_anon = true` and `is_temp = false` send in the events.
3. Edit an article on Test Wiki. Confirm that after publishing, you start to see `is_anon = true` and `is_temp = true`. If an event goes through the EditAttemptStep funnel, you will see a payload with `user_is_temp = true` and `is_anon = true`.
4. Continue clicking around and confirm you continue to see `is_anon = true` and `is_temp = true` logged.
5. Log into the app. After login and clicking around, confirm you now see `is_anon = false` and `is_temp = false`.
